src/stories/desktop/Header/Header.story.tsx

```diff
@@ -4,13 +4,18 @@ import { DesktopHeader } from '../../../desktop/DesktopHeader/DesktopHeader';
 import { LanguageDropdown } from '../../../common/LanguageDropdown/LanguageDropdown';
 import { SearchBar } from '../../../common/SearchBar/SearchBar';
 
-const LANGUAGES_ARR = [{short:'rus',long:'russian'},{short:'eng',long:'english'},{short:'est',long:'estonian'}]
+
+const LANGUAGES_ARR = [
```

Comment:
tha same as in `src/mobile/MobileHeaderBurger/MobileHeaderBurger.tsx`. We can move this array to common.